evap/staff/templates/staff_semester_view.html

```diff
@@ -4,8 +4,12 @@
 {% load evaluation_filters %}
 
 {% block content %}
-    {{ block.super }}
 
+    {{ block.super }}
+    <form id="form_activation_status" method="post" action="{% url 'rewards:semester_activation_edit' semester.id%}">
+        <input type="hidden" id="activation_status" name="activation_status" value="on"> {# FIXME #}
```

Comment:
I'd oppose having a fixme without any comment here. I would be (hesitantly) okay with a reference to an issue (`FIXME #123456`). Overall, I'm just in favor of adding the issue and not adding the fixme.  TODOs in code [have a tendency to stay there](https://github.com/id-Software/Quake-2/search?q=todo).

@niklasmohrin what do you think? (Follow-up to https://github.com/e-valuation/EvaP/pull/1837#discussion_r1039960441)

(repeated below in the javascript)

Reply:
No strong opinion, we can make an issue if you want